src/HEVM.hs

```diff
@@ -88,7 +87,7 @@ makeCtrCalldata (Interface _ decls) =
     mkArg :: Decl -> CalldataFragment
     mkArg (Decl typ x)  = symAbiArg (T.pack x) typ
     calldatas = fmap mkArg decls
-    (cdBuf, props) = combineFragments' calldatas 0 (EVM.AbstractBuf "txdata")
+    (cdBuf, props) = combineFragments' calldatas 0 (EVM.ConcreteBuf "")-- (EVM.AbstractBuf "txdata")
```

Comment:
why couldn't we use an `AbstractBuf` here again? would be cool to have some `TODO` comment explaining.

Reply:
The code is trying to compute the lengths of the buffer and it fails with an unexpected symbolic argument. I don't know what we can do about it thought in the future..

Reply:
```suggestion
    -- We need to use a concrete buf as a base here because hevm bails when trying to execute with an abstract buf
    -- This is because hevm ends up trying to execute a codecopy with a symbolic size, which is unsupported atm
    -- This is probably unsound, but theres not a lot we can do about it at the moment...
    (cdBuf, props) = combineFragments' calldatas 0 (EVM.ConcreteBuf "")-- (EVM.AbstractBuf "txdata")
```

Maybe a comment like this?